layers/+emacs/emacs-info/README.org

```diff
@@ -0,0 +1,10 @@
+#+TITLE: Emasc Info Imrprovements
+
+#+TAGS: emacs|layer
+
+* Description
+  This layer improves navigation for Emacs Info mode
+** Features:
+   - TODO: list features
+* Key bindings
```

Comment:
You are missing a mandatory Install block
```suggestion
* Install
To use this configuration layer, add it to your =~/.spacemacs=. You will need to
add =emacs-info= to the existing =dotspacemacs-configuration-layers= list in this
file.
```
After this the key bindings section should follow. Have a look at the suggested table of content for the right order and level please.